primefaces-selenium/primefaces-selenium-components/src/main/java/org/primefaces/selenium/component/Messages.java

```diff
@@ -39,6 +39,9 @@
  * Component wrapper for the PrimeFaces {@code p:messages}.
  */
 public abstract class Messages extends AbstractComponent {
+    public WebElement getClose(String severity) {
```

Comment:
Should we call it maybe getCloseElement?
maybe you can check it we call it in other cases, bit unsure

Reply:
Agreed `getCloseElement` would be more descriptive.

Reply:
`getCloseButton` seems to be the most reasonable choice, that name is already used in Dialog and StaticMessage